internal/lefthook/runner/runner.go

```diff
@@ -161,23 +161,24 @@ func (r *Runner) runScripts(dir string) {
 			continue
 		}
 
-		scriptPath := shellescape.Quote(filepath.Join(dir, file.Name()))
+		unquotedScriptPath := filepath.Join(dir, file.Name())
 
 		if r.hook.Parallel {
 			wg.Add(1)
 			go func(script *config.Script, path string, file os.FileInfo) {
 				defer wg.Done()
 				r.runScript(script, path, file)
-			}(script, scriptPath, file)
+			}(script, unquotedScriptPath, file)
 		} else {
-			r.runScript(script, scriptPath, file)
+			r.runScript(script, unquotedScriptPath, file)
 		}
 	}
 
 	wg.Wait()
 }
 
-func (r *Runner) runScript(script *config.Script, path string, file os.FileInfo) {
+func (r *Runner) runScript(script *config.Script, unquotedPath string, file os.FileInfo) {
+	quotedScriptPath := shellescape.Quote(unquotedPath)
```

Comment:
I think it's better to assign `quotedScriptPath` right before its usage (on line 211)

Reply:
I can do that!